app/controllers/users_controller.rb

```diff
@@ -136,61 +136,66 @@ def profile
     else
       @user = DrupalUser.find_by(name: params[:id])
       @profile_user = User.find_by(username: params[:id])
-      @title = @user.name
-      @notes = Node.research_notes
-                   .paginate(page: params[:page], per_page: 24)
-                   .order("nid DESC")
-                   .where(status: 1, uid: @user.uid)
-
-      if current_user && current_user.uid == @profile_user.uid
-        coauthor_nids = Node.joins(:node_tag).joins('LEFT OUTER JOIN term_data ON term_data.tid = community_tags.tid').select('node.*, term_data.*, community_tags.*').where(type: 'note', status: 3).where('term_data.name = (?)', "with:#{@profile_user.username}").collect(&:nid)
-        @drafts = Node.where('(nid IN (?) OR (status = 3 AND uid = ?))', coauthor_nids, @profile_user.uid).paginate(page: params[:page], per_page: 24)
-      end
-      @coauthored = @profile_user.coauthored_notes
-                                 .paginate(page: params[:page], per_page: 24)
-                                 .order('node_revisions.timestamp DESC')
-      @questions = @user.user.questions
-                             .order('node.nid DESC')
-                             .paginate(:page => params[:page], :per_page => 24)
-      @likes = (@user.liked_notes.includes(%i(tag comments)) + @user.liked_pages)
+      if !@user || !@profile_user
```

Comment:
I think you inserted 139-142 else git is showing without there is any need.
Great. Changes looks good to me.

Reply:
I just indented this. I added the condition only